setup.py

```diff
@@ -62,5 +62,5 @@
     ],
     test_suite='tests',
     tests_require=test_requirements,
-    url='https://github.com/revdotcom/rev_ai',
+    url='https://github.com/revdotcom/revai-python-sdk',
```

Comment:
Are we still going to use the "SDK" suffix for the name? As per Joshes earlier comment, this usually signifies there are a large number of compiled binaries or C code in this repo. We don't have any of those at the moment.